core/renderers/common/path_object.ts

```diff
@@ -40,12 +40,11 @@ export class PathObject implements IPathObject {
   constants: ConstantProvider;
   style: BlockStyle;
 
-  /**
-   * Highlight paths associated with connections.
-   *
-   * @protected
```

Comment:
Is taking this protected to private a breaking change? Or is this part of the unreleased code?

Reply:
This is unreleased!